dbus.c

```diff
@@ -266,36 +234,6 @@ PHP_DBUS_SETUP_TYPE_OBJ(int64,dbus_int64_t);
 PHP_DBUS_SETUP_TYPE_OBJ(uint64,dbus_uint64_t);
 PHP_DBUS_SETUP_TYPE_OBJ(double,double);
 
-struct _php_dbus_data_array {
-	int count;
-	void **data;
-	int size;
-};
-
-#define DBUS_SET_CONTEXT \
-	zval *object; \
-	object = getThis(); \
-   
-#define DBUS_FETCH_OBJECT	\
-	php_date_obj *obj;	\
-	DBUS_SET_CONTEXT; \
-	if (object) {	\
-		if (zend_parse_parameters_none() == FAILURE) {	\
-			return;	\
-		}	\
-	} else {	\
-		if (zend_parse_method_parameters(ZEND_NUM_ARGS() TSRMLS_CC, NULL, "O", &object, dbus_ce_dbus) == FAILURE) {	\
-			RETURN_FALSE;	\
-		}	\
-	}	\
-	obj = (php_dbus_obj *) zend_object_store_get_object(object TSRMLS_CC);	\
-
-#define DBUS_CHECK_INITIALIZED(member, class_name) \
```

Comment:
Is this macro/check no longer needed at all?

Reply:
it is not used at all even in master.